mkShapesRDF/shapeAnalysis/utils.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+
+def hist2array(h, flow=False, copy=True, include_sumw2=False):
+    nx = h.GetNbinsX() + 2
+    dtype = 0
+    if isinstance(h, ROOT.TH1D):
+        dtype = np.double
+    elif isinstance(h, ROOT.TH1F):
+        dtype = np.float
+    elif isinstance(h, ROOT.TH1I):
+        dtype = np.int
+    else:
+        print("Histogram of type", h, "is not supperted", file=sys.stderr)
```

Comment:
`supperted` -> `supported` (ignore this if you don't have time to update it and sorry if I am too nitpicking)